Makefile

```diff
@@ -3,8 +3,18 @@ ALL_GO_FILES=$$(find . -type f -name '*.go' -not -path "./vendor/*")
 .PHONY: setup
 setup:
 	go get -u golang.org/x/tools/cmd/goimports
+	go get -u github.com/alecthomas/gometalinter
```

Comment:
Should we be using something like gopkg.in to get the latest stable version?
https://gopkg.in/alecthomas/gometalinter.v2

Reply:
@scottgreenup : I think @amckague tried this recently and it complicated things

Reply:
It's worth a try. I didn't dig into what broke just that it did.